docs/events/timeline-ui-overview.asciidoc

```diff
@@ -202,7 +202,25 @@ The {ref}/esql.html[Elasticsearch Query Language ({esql})] provides a powerful w
 
 You can use {esql} in Timeline by opening the **{esql}** tab. From there, you can: 
 
-- Explore your events using the default query, or create a custom one. The default query searches documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>, then returns 10 events from the defined time range.  
+- Write an {esql} query to explore your events. For example, start with the following query, then iterate on it to tailor your results: 
++
+[source,esql]
+----
+FROM .alerts-security.alerts-default,apm-*-transaction*,auditbeat-*,endgame-*,filebeat-*,logs-*,packetbeat-*,traces-apm*,winlogbeat-*,-*elastic-cloud-logs-*
+| LIMIT 10
+| KEEP @timestamp, message, event.category, event.action, host.name, source.ip, destination.ip, user.name
+----
++
+This query does the following:
+
+** It starts by querying documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>.
+** Then, the query limits the output to the top 10 results.
+** Finally, it keeps the default `@timestamp`, `message`, `event.category`, `event.action`, `host.name`, `source.ip`, `destination.ip`, and `user.name` Timeline fields in the output.
++
+TIP: When querying large indices (for example, `logs-*`), performance can be impacted by the number of fields being returned in the output. If your query is taking a long time to complete, try using the {ref}/esql-commands.html#esql-keep[`KEEP`] command and specifying the fields you want returned. For example, add the clause `KEEP @timestamp, user.name` to the end of your query to specify that you only want the `@timestamp` and `user.name` fields returned.
```

Comment:
```suggestion
TIP: When querying indices that tend to be large (for example, `logs-*`), performance can be impacted by the number of fields being returned in the output. If your query is taking a long time to complete, try using the {ref}/esql-commands.html#esql-keep[`KEEP`] command and specifying the fields you want returned. For example, add the clause `KEEP @timestamp, user.name` to the end of your query to specify that you only want the `@timestamp` and `user.name` fields returned.
```

Reply:
@PhilippeOberti would you mind reviewing this as well? Thank you!

Reply:
the only thing that I would change is this part: `If your query is taking a long time to complete`

I feel like we want to try to avoid having customers running those query in the first place. So maybe phrase it a bit differently, like: `always try using the keep command unless necessary`?

Reply:
Ah, ok so frame using the `KEEP` clause as a best practice? How about this:

```suggestion
TIP: When querying indices that tend to be large (for example, `logs-*`), performance can be impacted by the number of fields being returned in the output. To optimize performance, use the {ref}/esql-commands.html#esql-keep[`KEEP`] command to specify the fields you want returned. For example, add the clause `KEEP @timestamp, user.name` to the end of your query to specify that you only want the `@timestamp` and `user.name` fields returned.
```

Reply:
Another option if "to optimize performance" is too large of a statement:

```suggestion
TIP: When querying indices that tend to be large (for example, `logs-*`), performance can be impacted by the number of fields being returned in the output. To reduce performance issues, use the {ref}/esql-commands.html#esql-keep[`KEEP`] command to specify fields you want returned. For example, add the clause `KEEP @timestamp, user.name` to the end of your query to specify that you only want the `@timestamp` and `user.name` fields returned.
```

Reply:
I like the first one better, but the second one works as well. I would even add `To optimize performance, we highly recommend using the ...` or something like that.

Reply:
```suggestion
TIP: When querying indices that tend to be large (for example, `logs-*`), performance can be impacted by the number of fields being returned in the output. To optimize performance, we recommend using the {ref}/esql-commands.html#esql-keep[`KEEP`] command to specify fields that you want returned. For example, add the clause `KEEP @timestamp, user.name` to the end of your query to specify that you only want the `@timestamp` and `user.name` fields returned.
```